test/run.sh

```diff
@@ -76,9 +86,15 @@ for dir in "${dirs[@]}"; do
         # Create the project here
         name=$(basename ${dir} .sh)
         set +e # For some reason the result here from head is not 0 even though we get the desired result
-        namespace=${name}-$(date -Ins | md5sum | tr -dc 'a-z0-9' | fold -w 6 | head -n 1)
+        if [[ "$OSTYPE" == "darwin"* ]]; then
+           # Mac OSX
+           namespace=${name}-$(date | md5sum | tr -dc 'a-z0-9' | fold -w 6 | head -n 1)
+
+        else
+           namespace=${name}-$(date -Ins | md5sum | tr -dc 'a-z0-9' | fold -w 6 | head -n 1)
+        fi
         set -e
-        oc new-project $namespace &> /dev/null
+        oc new-project $namespace
```

Comment:
I recall from Slack, there was an issue caused by the fact that `date` returned some different format on Mac, right? What I like for producing 'random' strings in bash is this:

```bash
foo=$(head /dev/urandom -c 512 | tr -dc a-z0-9 | head -c 16)
```

But I guess (from the code), the issue wasn't with the `date` command, but with the `readlink`.

Reply:
so from line 89-95 for mac it uses "date" but then for fedora it can use the param "Ins" - it is Ins that does not exist on mac @Jiri-Kremser